src/lib/app/identity.ts

```diff
@@ -48,11 +61,13 @@ export async function authenticate(id: string, secret: string): Promise<boolean>
     }
 }
 /**
- * Logout the current user
+ * Logout the current user and reset state
  * @returns void
  */
 export function logout(): void {
     authenticationData.set(undefined)
```

Comment:
this could also be done using the reset pattern as for the others `authenticationData.reset()`

Reply:
Done